CORE-6698 Vnode reconciler goes down

kyriathar opened the pull request:
The problem is, Vnode reconciler goes down.

Vnode reconciler component depends on `VirtualNodeInfoWriterComponentImpl` (`VirtualNodeInfoWriteService` is used as the Kafka writer in Vnode info reconciliation). 

In `VirtualNodeInfoWriterComponentImpl.onConfigChangedEventReceived` we would switch coordinator status to `DOWN` every time we would receive a `ConfigChangedEvent` to mark that component is down during resources re-creation. However, that is propagated to components depending on this one and therefore to Vnode reconciler.

- removes `VirtualNodeInfoWriterComponentImpl` switching to `DOWN` every time we receive a `ConfigChangedEvent`.
- improves logging for Vnode reconciler readers and writer.

kyriathar commented:
Please hold on with review as an unwanted commit appeared. Will close and re-open PR against [release/os/5.0-DevPreview2](https://github.com/corda/corda-runtime-os/tree/release/os/5.0-DevPreview2)

vkolomeyko commented:
> Please hold on with review as an unwanted commit appeared. Will close and re-open PR against [release/os/5.0-DevPreview2](https://github.com/corda/corda-runtime-os/tree/release/os/5.0-DevPreview2)

Potentially possible to fix-up this PR by force pushing to your branch: `kyriakos/CORE-6698-vnode-reconciler-goes-down`

kyriathar commented:
>>Please hold on with review as an unwanted commit appeared. Will close and re-open PR against [release/os/5.0-DevPreview2](https://github.com/corda/corda-runtime-os/tree/release/os/5.0-DevPreview2)

>Potentially possible to fix-up this PR by force pushing to your branch: kyriakos/CORE-6698-vnode-reconciler-goes-down

Thank you @vkolomeyko but unfortunately that didn't work.

kyriathar commented:
Fixed now, it's ready for review.

vkolomeyko reviewed (dismissed):
LGTM

kyriathar-r3 committed:
CORE-6698 Vnode reconciler goes down (#2092)

The problem was, Vnode reconciler was going down.

Vnode reconciler component depends on `VirtualNodeInfoWriterComponentImpl` (`VirtualNodeInfoWriteService` is used as the Kafka writer in Vnode info reconciliation).

In `VirtualNodeInfoWriterComponentImpl.onConfigChangedEventReceived` we would switch coordinator status to DOWN every time we would receive a `ConfigChangedEvent` to mark that component is down during resources re-creation. However, that is propagated to components depending on this one and therefore to Vnode reconciler.

* removes `VirtualNodeInfoWriterComponentImpl` switching to DOWN every time we receive a `ConfigChangedEvent`.

* improves logging for Vnode reconciler readers and writer.

vkolomeyko reviewed (approved):
LGTM
